src/anomalib/models/dsr/torch_model.py

```diff
@@ -153,6 +153,8 @@ def forward(self, batch: Tensor, anomaly_map_to_generate: Tensor | None = None)
                     out_mask_sm[:, 1:, :, :], 21, stride=1, padding=21 // 2
                 ).detach()
                 image_score = torch.amax(out_mask_averaged, dim=(2, 3)).squeeze()
+                if image_score.size() == torch.Size([]):
```

Comment:
Would it be possible to add a comment here to explain why this is needed. Mostly for a future reference.

Reply:
Fixed!